src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,134 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const currentSex = searchParams.get('sex') || null;
+
+  type ParamsType = {
+    sort: string | null,
+    order: string | null,
+    sex: 'm' | 'f' | null,
+    query: string | null,
+    centuries: number | null,
+  };
+
+  function setSearchWith(params: Partial<ParamsType>) {
+    const currentSort = searchParams.get('sort') || '';
+    const currentOrder = searchParams.get('order') || null;
+
+    const updatedParams: ParamsType = {
+      sort: !currentSort ? null : currentSort,
```

Comment:
This can be written more simply
```suggestion
      sort: currentSort || null,
```
Fix this in other places

Reply:
I reimplement that 🤝 (i hope)